tests/test_scan.py

```diff
@@ -345,7 +345,7 @@ def scan_fn(mus_t, sigma_t, Y_t_val, S_t_val, Gamma_t):
     assert np.allclose(y_logp_val, y_logp_ref_val)
 
 
-@aesara.config.change_flags(compute_test_value="raise")
```

Comment:
Ah... we should not delete this previous decorator. The xfail still works with it no?

Reply:
Oops... I made the edits too hastily. I corrected this now